spec/spec_helper.rb

```diff
@@ -18,6 +18,7 @@
 require_relative 'simplecov_helper'
 require 'bigcommerce/prometheus'
 require 'pry'
+require 'rails'
```

Comment:
This gem is used in non-rails systems, so we shouldn't pull in this dependency here.